rpp-charter.md

```diff
@@ -60,18 +60,27 @@ for the promotion of work to the IESG.
 This working group will find consensus on a set of core requirements for RPP.
 In alignment with the [IESG's Statement on Support Documents dated 2023-08-24](https://datatracker.ietf.org/doc/statement-iesg-support-documents-in-ietf-working-groups-20230824/),
 these requirements will be publicly available on the working group's wiki.
+Analysis of the functionality included and commonly used in core EPP for domain names, 
+hosts and contacts (RFC5730,5731,5732,5733) may be created as input material for the requirements. 
```

Comment:
I like the rearrangement but noticed the change from "will" to "may".
The discussion on the mailing list so far was rather going into direction that such analysis will be created as an input.

Reply:
> If nobody is willing to do the analysis or there is no agreement on that analysis, the working group should not stall. This is an area of potential rabbit hole exploring. Thanks for noting this.

I think if nobody would be willing to, it would indicate other problems in the working group. The conclusion of the discussion so far was that without this input we may not be able to validate if requirements and architecture are actually correct and complete to fulfil other goals and deliverables.
The text does not mean that there must be only one or more, in which form and even not if there must be a consensus around the analysis. And this is meant like this. More perspectives are typically better on an early stage.

The requirements is where we will be looking after consensus. With analysis done we may expect better quality of requirements coming out of the process.